src/Model.php

```diff
@@ -710,6 +710,7 @@ public function addCondition($field, $operator = null, $value = null)
 
         if ($f) {
             $f->system = true;
+            $f->ui['editable'] = false;
```

Comment:
I see and it's passed into field here: https://github.com/atk4/data/blob/develop/src/Reference_One.php#L84

I guess we shouldn't set the default there. Do you think it would work if UI would be null?